packages/node_modules/nav-frontend-ekspanderbartpanel/package.json

```diff
@@ -12,23 +12,22 @@
     "type": "git",
     "url": "git+https://github.com/navikt/nav-frontend-moduler.git"
   },
-  "dependencies": {
-    "nav-frontend-typografi": "0.2.1",
-    "nav-frontend-ekspanderbartpanel-style": "0.2.1"
-  },
   "peerDependencies": {
     "react": "^15.4.2",
-    "classnames": "^2.2.5",
     "prop-types": "^15.5.10",
-    "react-collapse": "^2.3.3"
+    "classnames": "^2.2.5",
+    "react-collapse": "^4.0.2",
+    "nav-frontend-typografi": "^0.1.12",
+    "nav-frontend-ekspanderbartpanel-style": "^0.1.10"
```

Comment:
Disse to ser ut til å ha feil versjon, skal ha ^0.2.1 vel? Ligger også under devDeps, mulig de kan fjernes derifra?